PlataformaMusical.h

```diff
@@ -0,0 +1,70 @@
+﻿#pragma once
+#pragma once
+#include <iostream>
+#include <string>
+#include "Artista.h"
+#include "Cancion.h"
+#include "ArrayGenerica.h"
+#include "ElementoYaExistenteException.h"
+#include "ElementoNoExistenteException.h"
+using namespace std;
+
+
+/*
+Clase Padre: Artista (attr: username, generoMusical )
+○ Clase hija: Cantante (attr adicionales: tonoVoz )
+○ Clase hija: Compositor (attr adicionales: instrumentoPrincipal )
+○ Nota: En este caso se debe aplicar herencia, ya que Cantante y Compositor comparten características comunes de la clase Artista.
+● Cancion (attr: titulo, Artista* compositor, Artista* cantante)
+● PlataformaMusical (attr: nombrePlataforma, listaCanciones, listaArtistas )
+*/
+
+class PlataformaMusical {
+protected:
+	string nombrePlataforma;
+	ArrayGenerica<Cancion>* listaCanciones;
+	ArrayGenerica<Artista>* listaArtistas;
+public:
+	PlataformaMusical(string _nombrePlataforma, int capCanciones, int capArtistas) {
+		this->nombrePlataforma = _nombrePlataforma;
+		this->listaArtistas = new ArrayGenerica<Artista>(capArtistas);
+		this->listaCanciones = new ArrayGenerica<Cancion>(capCanciones);
+	}
+	void mostrar(string valor) {
+		if (listaArtistas->buscarElemento(valor, Artista::comparador))
+		{
+			cout << "nombrePlataforma : " << nombrePlataforma << endl;
+			listaArtistas->mostrar();
+			listaCanciones->mostrar();
+		}
+		
+	}
+	/*--------------------------------------*/
+	void agregarArtista(string valor, Artista* nuevoArtista) {
+		if (listaArtistas!=nullptr) {

```

Comment:
El atributo valor esta sobrando, en caso de que necesites el nombre, podrias utilizar el metodo getNombre. Por ejemplo 

```suggestion
	void agregarArtista(Artista* nuevoArtista) {
	string nombreArtsita = nuevoArtista->getNombre();
		if (listaArtistas!=nullptr) {

```